tools/src/main/java/org/wildfly/swarm/tools/BuildTool.java

```diff
@@ -204,6 +204,10 @@ public File build(String baseName, Path dir) throws Exception {
 
 
     public void repackageWar(File file) throws IOException {
+        if (!filterWebInfLib) {
+            return;
+        }
+
         this.log.info("Repackaging .war: " + file);
```

Comment:
In this method, the file is first moved, then overwritten by the same content if `filterWebInfLib` is marked as false. Hence the suggestion for improvement. Please do share your thoughts about it.

Reply:
I think originally, there was no `filterWebInfLib` setting, we always filtered. `filterWebInfLib` defaults to `true`, if I remember correctly.

This change makes sense, it's better than `Archive repackaged = this.filterWebInfLib ? new WebInfLibFilteringArchive(original, this.dependencyManager) : original;`. Perhaps we should revert that back to `Archive repackaged = new WebInfLibFilteringArchive(original, this.dependencyManager)`

Reply:
You are correct. The default value for `filterWebInfLib` is set to true. I looked at the [WebInfLibFilteringArchive](https://github.com/thorntail/thorntail/blob/master/tools/src/main/java/org/wildfly/swarm/tools/WebInfLibFilteringArchive.java#L48) and the only thing that it seems to do is remove the `thorntail-runner` jar from the library assets.

I can make the change to the conditional as well. Let me know if you have other review comments, if not I will update this PR in the evening which will incorporate the above comment.

Reply:
I don't have any other comment.

There's a Windows test failure, but it seems unrelated, so feel free to go ahead.

Reply:
Perfect timing!